### PR TITLE
Added the Dotnet Example for the Delete Results API

### PR DESCRIPTION
**Pull request by Madhan-Reddy-ni**
**Justification**
In order to represent how to use the APIs created in the TestMonitor service in the SystemLink Enterprise version, there is a need to create examples that will show how to call the APIs, what details are needed to be sent with the API call and etc.

**Implementation**
In this example I have represented how to use the Delete Result API and Delete Results API

**Testing**
Manual testing completed

### CODE REVIEW

File: DotNet Examples/Test Monitor/Delete Results/Extensions.cs [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Delete_Results
```
Comment (ramganesh-ni) [inline DotNet Examples/Test Monitor/Delete Results/Extensions.cs]: Please update namespace naming similar to the `Results` example.

File: DotNet Examples/Test Monitor/Results/Results.csproj [hunk 2]
```diff
@@ -11,4 +11,7 @@
 		</PackageReference>
 		<PackageReference Include="NationalInstruments.SystemLink.Clients.TestMonitor" Version="2.1.1" />
 	</ItemGroup>
+	<ItemGroup>
```
Comment (ramganesh-ni) [inline DotNet Examples/Test Monitor/Results/Results.csproj:14]: Why is this file included? Why do we need changes to the Results project file for Delete example

File: DotNet Examples/Test Monitor/Delete Results/Extensions.cs [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+            response.EnsureSuccessStatusCode();
+            return response;
+        }
+
+        private static HttpClient GetHttpClient(IHttpConfiguration configuration)
+        {
+            var client = new HttpClient();
+            client.BaseAddress = new Uri(configuration.ServerUri.AbsoluteUri);
+            client.DefaultRequestHeaders.Add("x-ni-api-key", configuration.ApiKeys["x-ni-api-key"]);
+            return client;
```
Comment (ramganesh-ni) [inline DotNet Examples/Test Monitor/Delete Results/Extensions.cs]:
```suggestion

            return client;
```

File: DotNet Examples/Test Monitor/Delete Results/Extensions.cs [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+{
+    static class Extensions
+    {
+        public static HttpResponseMessage DeleteResultAsync(this TestDataManager testDataManager, IHttpConfiguration configuration, string resultId, bool? deleteSteps = true)
+        {
+            if (resultId == null)
+            {
+                throw new ApiException("Missing required parameter 'resultId' when calling ResultsApi->DeleteResultV2");
+            }
+            string path = $"nitestmonitor/v2/results/{resultId}?deleteSteps={deleteSteps}";
```
Comment (ramganesh-ni) [inline DotNet Examples/Test Monitor/Delete Results/Extensions.cs]: Can we move base URIs as constants ?

File: DotNet Examples/Test Monitor/Delete Results/Extensions.cs [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Delete_Results
+{
+    static class Extensions
```
Comment (ramganesh-ni) [inline DotNet Examples/Test Monitor/Delete Results/Extensions.cs]: Needs better naming as this extension specific to TestDataManager Delete

File: DotNet Examples/Test Monitor/Delete Results/Extensions.cs [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+            }
+            string path = $"nitestmonitor/v2/delete-results";
+
+            var content = new
+            {
+                Ids = resultIds.ToArray(),
+                DeleteSteps = deleteSteps
+            };
+
+            var body = new StringContent(JsonConvert.SerializeObject(content), Encoding.UTF8, "application/json");
```
Comment (ramganesh-ni) [inline DotNet Examples/Test Monitor/Delete Results/Extensions.cs]: can we tidy this up a bit and move implementation details to the private method?

**Review by ramganesh-ni — CHANGES_REQUESTED** [review]

File: DotNet Examples/Test Monitor/Delete Results/Program.cs [hunk 7]
```diff
@@ -0,0 +1,52 @@
+﻿using NationalInstruments.SystemLink.Clients.Examples.TestMonitor.DeleteResults;
+using NationalInstruments.SystemLink.Clients.TestMonitor;
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+
+namespace NationalInstruments.SystemLink.Clients.Examples.TestMonitor
+{
+    class Program
```
Comment (mohan-ni) [inline DotNet Examples/Test Monitor/Delete Results/Program.cs]: Can use **DeleteResults** as the class name and file name instead of **Program**.

File: DotNet Examples/Test Monitor/Delete Results/Delete Results.csproj [hunk 8]
```diff
@@ -0,0 +1,17 @@
+<Project Sdk="Microsoft.NET.Sdk">
```
Comment (mohan-ni) [inline DotNet Examples/Test Monitor/Delete Results/Delete Results.csproj]: Rename the project file and folder into **DeleteResults**

**Comment by mohan-ni** [comment]
Add **Readme** file

File: DotNet Examples/Test Monitor/Delete Results/Program.cs [hunk 9]
```diff
@@ -0,0 +1,52 @@
+﻿using NationalInstruments.SystemLink.Clients.Examples.TestMonitor.DeleteResults;
+using NationalInstruments.SystemLink.Clients.TestMonitor;
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+
+namespace NationalInstruments.SystemLink.Clients.Examples.TestMonitor
```
Comment (mohan-ni) [inline DotNet Examples/Test Monitor/Delete Results/Program.cs]:
```suggestion
namespace NationalInstruments.SystemLink.Clients.Examples.TestMonitor.DeleteResults
```

**Comment by mohan-ni** [comment]
Update the code to create results, and wait for the user's response before deleting the results.

File: DotNet Examples/Test Monitor/Delete Results/TestDataManagerDeleteExtensions.cs [hunk 10]
```diff
@@ -0,0 +1,86 @@
...
+using System.Linq;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Reflection.Metadata;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace NationalInstruments.SystemLink.Clients.Examples.TestMonitor.DeleteResults
+{
+    static class TestDataManagerDeleteExtensions
```
Comment (mohan-ni) [inline DotNet Examples/Test Monitor/Delete Results/TestDataManagerDeleteExtensions.cs]: Add documentation to the methods

File: DotNet Examples/Test Monitor/Delete Results/Program.cs (hunk 7 above)
Comment (Madhan-Reddy-ni) [reply DotNet Examples/Test Monitor/Delete Results/Program.cs]: updated the class name and file name

File: DotNet Examples/Test Monitor/Delete Results/Delete Results.csproj (hunk 8 above)
Comment (Madhan-Reddy-ni) [reply DotNet Examples/Test Monitor/Delete Results/Delete Results.csproj]: updated the project name

File: DotNet Examples/Test Monitor/Delete Results/Program.cs (hunk 9 above)
Comment (Madhan-Reddy-ni) [reply DotNet Examples/Test Monitor/Delete Results/Program.cs]: applied the suggestion

**Comment by Madhan-Reddy-ni** [comment]
> Add **Readme** file

Added the Readme file

**Comment by Madhan-Reddy-ni** [comment]
> Update the code to create results, and wait for the user's response before deleting the results.

Applied this suggestion

File: DotNet Examples/Test Monitor/Delete Results/Extensions.cs (hunk 1 above)
Comment (Madhan-Reddy-ni) [reply DotNet Examples/Test Monitor/Delete Results/Extensions.cs]: updated the namespace

File: DotNet Examples/Test Monitor/Delete Results/Extensions.cs (hunk 3 above)
Comment (Madhan-Reddy-ni) [reply DotNet Examples/Test Monitor/Delete Results/Extensions.cs]: applied the suggestion

File: DotNet Examples/Test Monitor/Delete Results/Extensions.cs (hunk 4 above)
Comment (Madhan-Reddy-ni) [reply DotNet Examples/Test Monitor/Delete Results/Extensions.cs]: moved the base url to constants

File: DotNet Examples/Test Monitor/Delete Results/Extensions.cs (hunk 5 above)
Comment (Madhan-Reddy-ni) [reply DotNet Examples/Test Monitor/Delete Results/Extensions.cs]: Updated the name to `TestDataManagerDeleteExtensions`

File: DotNet Examples/Test Monitor/Delete Results/Extensions.cs (hunk 6 above)
Comment (Madhan-Reddy-ni) [reply DotNet Examples/Test Monitor/Delete Results/Extensions.cs]: Applied this suggestion by creating private methods

File: DotNet Examples/README.md [hunk 11]
```diff
@@ -38,4 +38,5 @@ API Examples
 ------------
 ### [Test Monitor](testmonitor)
 
-- [Results](testmonitor/results): Demonstrates how to use the SystemLink Test Monitor API to publish test results to the server.
\ No newline at end of file
+- [Results](testmonitor/results): Demonstrates how to use the SystemLink Test Monitor API to publish test results to the server.
+- [DeleteResults](testmonitor/DeleteResults): Demonstrates hw to use the SystemLink Test Monitor API to create and delete test results.
```
Comment (mohan-ni) [inline DotNet Examples/README.md]:
```suggestion
- [DeleteResults](testmonitor/DeleteResults): Demonstrates how to use the SystemLink Test Monitor API to create and delete test results.
```

File: DotNet Examples/Test Monitor/README.md [hunk 12]
```diff
@@ -25,4 +25,5 @@ Test Monitor Examples
 ------------
 
 - [Results](results): Demonstrates how to use the SystemLink Test Monitor API to create test results and steps.
+- [DeleteResults](DeleteResults): Demonstrates hw to use the SystemLink Test Monitor API to create and delete test results.
```
Comment (mohan-ni) [inline DotNet Examples/Test Monitor/README.md]:
```suggestion
- [DeleteResults](DeleteResults): Demonstrates how to use the SystemLink Test Monitor API to create and delete test results.
```

**Review by mohan-ni — APPROVED** [review]

File: DotNet Examples/Test Monitor/Delete Results/TestDataManagerDeleteExtensions.cs (hunk 10 above)
Comment (Madhan-Reddy-ni) [reply DotNet Examples/Test Monitor/Delete Results/TestDataManagerDeleteExtensions.cs]: Documentation added

File: DotNet Examples/README.md (hunk 11 above)
Comment (Madhan-Reddy-ni) [reply DotNet Examples/README.md]: applied

File: DotNet Examples/Test Monitor/README.md (hunk 12 above)
Comment (Madhan-Reddy-ni) [reply DotNet Examples/Test Monitor/README.md]: Applied

File: DotNet Examples/Test Monitor/DeleteResults/DeleteResults.cs [hunk 13]
```diff
@@ -0,0 +1,60 @@
+﻿using NationalInstruments.SystemLink.Clients.TestMonitor;
```
Comment (ramganesh-ni) [inline DotNet Examples/Test Monitor/DeleteResults/DeleteResults.cs]: As we have two examples for Results now - can we make the folder structure as follow - 

`````
Test Monitor
--> Results 
------> Create Results
------> Delete Results

`````

File: DotNet Examples/README.md [hunk 14]
```diff
@@ -38,4 +38,5 @@ API Examples
 ------------
 ### [Test Monitor](testmonitor)
 
-- [Results](testmonitor/results): Demonstrates how to use the SystemLink Test Monitor API to publish test results to the server.
\ No newline at end of file
+- [Results](testmonitor/results): Demonstrates how to use the SystemLink Test Monitor API to publish test results to the server.
```
Comment (ramganesh-ni) [inline DotNet Examples/README.md]: Can we start referring this as  `Create Results` Example

File: DotNet Examples/Test Monitor/README.md [hunk 15]
```diff
@@ -25,4 +25,5 @@ Test Monitor Examples
 ------------
 
 - [Results](results): Demonstrates how to use the SystemLink Test Monitor API to create test results and steps.
```
Comment (ramganesh-ni) [inline DotNet Examples/Test Monitor/README.md]: Does this example also handle creation of steps? Then please consider renaming the title here accordingly

File: DotNet Examples/Test Monitor/DeleteResults/DeleteResults.csproj [hunk 16]
```diff
@@ -0,0 +1,17 @@
...
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <RootNamespace>Delete_Results</RootNamespace>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="NationalInstruments.SystemLink.Clients.TestMonitor" Version="2.1.1" />
+  </ItemGroup>
+
+	<ItemGroup>
```
Comment (ramganesh-ni) [inline DotNet Examples/Test Monitor/DeleteResults/DeleteResults.csproj]: Please indent these lines properly

File: DotNet Examples/Test Monitor/DeleteResults/TestDataManagerDeleteExtensions.cs [hunk 17]
```diff
@@ -0,0 +1,126 @@
...
+            return CallDeleteResultApi(configuration, path);
+        }
+
+        /// <summary>
+        /// calls the delete result API
+        /// </summary>
+        /// <param name="configuration">A HttpConfiguration object containing base url and api_key</param>
+        /// <param name="requestUri">Uri route</param>
+        /// <returns>Http reponse message after calling the API</returns>
+        private static HttpResponseMessage CallDeleteResultApi(IHttpConfiguration configuration, string requestUri)
```
Comment (ramganesh-ni) [inline DotNet Examples/Test Monitor/DeleteResults/TestDataManagerDeleteExtensions.cs]: Move below Public methods

**Review by ramganesh-ni — APPROVED** [review]

File: DotNet Examples/Test Monitor/DeleteResults/TestDataManagerDeleteExtensions.cs (hunk 17 above)
Comment (Madhan-Reddy-ni) [reply DotNet Examples/Test Monitor/DeleteResults/TestDataManagerDeleteExtensions.cs]: Implemented

File: DotNet Examples/Test Monitor/DeleteResults/DeleteResults.csproj (hunk 16 above)
Comment (Madhan-Reddy-ni) [reply DotNet Examples/Test Monitor/DeleteResults/DeleteResults.csproj]: Implemented

File: DotNet Examples/Test Monitor/DeleteResults/DeleteResults.cs (hunk 13 above)
Comment (Madhan-Reddy-ni) [reply DotNet Examples/Test Monitor/DeleteResults/DeleteResults.cs]: Renamed the `Results` folder to `CreateResultsAndSteps`

File: DotNet Examples/Test Monitor/README.md (hunk 15 above)
Comment (Madhan-Reddy-ni) [reply DotNet Examples/Test Monitor/README.md]: Renamed the project to `CreateResultsAndSteps`

File: DotNet Examples/README.md (hunk 14 above)
Comment (Madhan-Reddy-ni) [reply DotNet Examples/README.md]: Referring as `CreateResultsAndSteps`

File: DotNet Examples/TestMonitor/DeleteResults/DeleteResults.cs [hunk 18]
```diff
@@ -8,53 +9,66 @@ class DeleteResults
...
+                Console.WriteLine("\nResults has been deleted successfully");
+            }
+            catch(ApiException exception)
             {
-                resultData.ProgramName = resultData.ProgramName + i;
-                var result = testDataManager.CreateResult(resultData);
-                testResultIds.Add(result.Id);
-                Console.WriteLine($"{result.Id}");
+                Console.WriteLine(exception.Message);
+                Console.WriteLine("The given Url or the API key was wrong, please run the example with the correct Url and api_key.");
```
Comment (mohan-ni) [inline DotNet Examples/TestMonitor/DeleteResults/DeleteResults.cs]: Can we update the first error message as 
"The given URL or API key might be invalid or the Server might be down. Please re-check the URL, API key, and the Server."

**Review by mohan-ni — APPROVED** [review]

File: DotNet Examples/TestMonitor/DeleteResults/DeleteResults.cs (hunk 18 above)
Comment (Madhan-Reddy-ni) [reply DotNet Examples/TestMonitor/DeleteResults/DeleteResults.cs]: Updated the error message

File: DotNet Examples/TestMonitor/DeleteResults/DeleteResults.cs [hunk 19]
```diff
@@ -0,0 +1,74 @@
...
+                {
+                    Operator = "John Smith",
+                    ProgramName = "Power Test",
+                    Status = new Status(StatusType.Running),
+                    SerialNumber = Guid.NewGuid().ToString(),
+                    PartNumber = "NI-ABC-123-PWR1"
+                };
+                // Create the test result.
+                var testResult = testDataManager.CreateResult(resultData);
+                Console.WriteLine($"The test result has been under part number={testResult.Data.PartNumber} with Id = {testResult.Id}");
```
Comment (ramganesh-ni) [inline DotNet Examples/TestMonitor/DeleteResults/DeleteResults.cs]:
```suggestion
                Console.WriteLine($"New test result is created under part number={testResult.Data.PartNumber} with Id = {testResult.Id}");
```

File: DotNet Examples/TestMonitor/DeleteResults/DeleteResults.cs [hunk 20]
```diff
@@ -0,0 +1,74 @@
...
+                Console.WriteLine("Press any key to delete the result");
+                Console.ReadKey();
+
+                // Delete the test result
+                testDataManager.DeleteResultAsync(configuration, testResult.Id);
+                Console.WriteLine($"\nThe test result with Id = {testResult.Id} has been deleted");
+
+                // Create multiple test results
+                Console.WriteLine("\nCreating multiple test results.\nResult Ids has been listed down below");
```
Comment (ramganesh-ni) [inline DotNet Examples/TestMonitor/DeleteResults/DeleteResults.cs]:
```suggestion
                Console.WriteLine("\nCreating multiple test results.\nResult Ids are listed below");
```

File: DotNet Examples/TestMonitor/DeleteResults/DeleteResults.cs [hunk 21]
```diff
@@ -0,0 +1,74 @@
...
+                };
+                // Create the test result.
+                var testResult = testDataManager.CreateResult(resultData);
+                Console.WriteLine($"The test result has been under part number={testResult.Data.PartNumber} with Id = {testResult.Id}");
+                Console.WriteLine("Press any key to delete the result");
+                Console.ReadKey();
+
+                // Delete the test result
+                testDataManager.DeleteResultAsync(configuration, testResult.Id);
+                Console.WriteLine($"\nThe test result with Id = {testResult.Id} has been deleted");
```
Comment (ramganesh-ni) [inline DotNet Examples/TestMonitor/DeleteResults/DeleteResults.cs]:
```suggestion
                Console.WriteLine($"\nThe test result with Id = {testResult.Id} is deleted");
```

File: DotNet Examples/TestMonitor/DeleteResults/DeleteResults.cs [hunk 22]
```diff
@@ -0,0 +1,74 @@
...
+                for (int i = 0; i < 5; i++)
+                {
+                    resultData.ProgramName = resultData.ProgramName + i;
+                    var result = testDataManager.CreateResult(resultData);
+                    testResultIds.Add(result.Id);
+                    Console.WriteLine($"{result.Id}");
+                }
+
+                // Delete the test results
+                Console.WriteLine("\nThe multiple results has been created successfully");
```
Comment (ramganesh-ni) [inline DotNet Examples/TestMonitor/DeleteResults/DeleteResults.cs]:
```suggestion
                Console.WriteLine("\nTest results created successfully");
```

File: DotNet Examples/TestMonitor/DeleteResults/DeleteResults.cs [hunk 23]
```diff
@@ -0,0 +1,74 @@
...
+                Console.WriteLine("Please any key to delete these results");
```
Comment (ramganesh-ni) [inline DotNet Examples/TestMonitor/DeleteResults/DeleteResults.cs]:
```suggestion
                Console.WriteLine("Press any key to delete these results");
```

File: DotNet Examples/TestMonitor/DeleteResults/DeleteResults.cs [hunk 24]
```diff
@@ -0,0 +1,74 @@
...
+                Console.ReadKey();
+                testDataManager.DeleteResultsAsync(configuration, testResultIds);
+                Console.WriteLine("\nResults has been deleted successfully");
```
Comment (ramganesh-ni) [inline DotNet Examples/TestMonitor/DeleteResults/DeleteResults.cs]:
```suggestion
                Console.WriteLine("\nResults deleted successfully");
```

File: DotNet Examples/TestMonitor/DeleteResults/DeleteResults.cs [hunk 25]
```diff
@@ -0,0 +1,74 @@
...
+            }
+            catch(ApiException exception)
+            {
+                Console.WriteLine(exception.Message);
+                Console.WriteLine("The given URL or API key might be invalid or the Server might be down. Please re-check the URL, API key, and the Server.");
```
Comment (ramganesh-ni) [inline DotNet Examples/TestMonitor/DeleteResults/DeleteResults.cs]:
```suggestion
                Console.WriteLine("The given URL or API key might be invalid or the server might be down. Please try again after verifying the server is up and the URL or API key is valid");
```

File: DotNet Examples/TestMonitor/DeleteResults/TestDataManagerDeleteExtensions.cs [hunk 26]
```diff
@@ -0,0 +1,134 @@
...
+        private const string DeleteResultsPath = "nitestmonitor/v2/delete-results";
+        private const string DeleteResultPath = "nitestmonitor/v2/results";
+
+        /// <summary>
+        /// Deletes the test result
+        /// </summary>
+        /// <param name="testDataManager"></param>
+        /// <param name="configuration">HttpConfiguration containg the url and API key</param>
+        /// <param name="resultId">test result ID which needs to be deleted</param>
+        /// <param name="deleteSteps">A boolen representing whether to delete the associated steps or not</param>
```
Comment (ramganesh-ni) [inline DotNet Examples/TestMonitor/DeleteResults/TestDataManagerDeleteExtensions.cs]:
```suggestion
        /// <param name="deleteSteps">A boolean representing whether to delete the associated steps or not</param>
```

File: DotNet Examples/TestMonitor/DeleteResults/TestDataManagerDeleteExtensions.cs [hunk 27]
```diff
@@ -0,0 +1,134 @@
...
+        /// <exception cref="ApiException"></exception>
+        public static HttpResponseMessage DeleteResultAsync(this TestDataManager testDataManager, IHttpConfiguration configuration, string resultId, bool? deleteSteps = true)
+        {
+            if (resultId == null)
+            {
+                throw new ApiException("Missing required parameter 'resultId' when calling ResultsApi->DeleteResultV2");
+            }
+            string path = $"{DeleteResultPath}/{resultId}?deleteSteps={deleteSteps}";
+
+            return CallDeleteResultApi(configuration, path);
```
Comment (ramganesh-ni) [inline DotNet Examples/TestMonitor/DeleteResults/TestDataManagerDeleteExtensions.cs]: Can we move the HTTP call details into a new file, something like `Client` facade?

File: DotNet Examples/TestMonitor/DeleteResults/TestDataManagerDeleteExtensions.cs [hunk 28]
```diff
@@ -0,0 +1,134 @@
...
+            return CallDeleteResultsApi(configuration, body);
+        }
+
+        /// <summary>
+        /// calls the delete result API
+        /// </summary>
+        /// <param name="configuration">A HttpConfiguration object containing base url and api_key</param>
+        /// <param name="requestUri">Uri route</param>
+        /// <returns>Http reponse message after calling the API</returns>
+        private static HttpResponseMessage CallDeleteResultApi(IHttpConfiguration configuration, string requestUri)
```
Comment (ramganesh-ni) [inline DotNet Examples/TestMonitor/DeleteResults/TestDataManagerDeleteExtensions.cs]: Name these methods better, it is not necessary to include these prefixes and rather use just verbs to explain the action

**Review by ramganesh-ni — APPROVED** [review]
I left a few comments, Please tidy them up by moving the HTTP client call details into a new file. Also, verify the texts you print to the console and ensure they are grammatically accurate. 

cc: @mohan-ni

File: DotNet Examples/TestMonitor/DeleteResults/DeleteResults.cs (hunk 19 above)
Comment (Madhan-Reddy-ni) [reply DotNet Examples/TestMonitor/DeleteResults/DeleteResults.cs]: Changed the message

File: DotNet Examples/TestMonitor/DeleteResults/DeleteResults.cs (hunk 21 above)
Comment (Madhan-Reddy-ni) [reply DotNet Examples/TestMonitor/DeleteResults/DeleteResults.cs]: Updated the message

File: DotNet Examples/TestMonitor/DeleteResults/DeleteResults.cs (hunk 22 above)
Comment (Madhan-Reddy-ni) [reply DotNet Examples/TestMonitor/DeleteResults/DeleteResults.cs]: Updated the message

File: DotNet Examples/TestMonitor/DeleteResults/DeleteResults.cs (hunk 24 above)
Comment (Madhan-Reddy-ni) [reply DotNet Examples/TestMonitor/DeleteResults/DeleteResults.cs]: Updated the message

File: DotNet Examples/TestMonitor/DeleteResults/DeleteResults.cs (hunk 25 above)
Comment (Madhan-Reddy-ni) [reply DotNet Examples/TestMonitor/DeleteResults/DeleteResults.cs]: Updated the message

File: DotNet Examples/TestMonitor/DeleteResults/TestDataManagerDeleteExtensions.cs (hunk 26 above)
Comment (Madhan-Reddy-ni) [reply DotNet Examples/TestMonitor/DeleteResults/TestDataManagerDeleteExtensions.cs]: Updated the message

File: DotNet Examples/TestMonitor/DeleteResults/TestDataManagerDeleteExtensions.cs (hunk 27 above)
Comment (Madhan-Reddy-ni) [reply DotNet Examples/TestMonitor/DeleteResults/TestDataManagerDeleteExtensions.cs]: Created the separate file for HTTP calls named as TestMonitorHttpClient

File: DotNet Examples/TestMonitor/DeleteResults/TestDataManagerDeleteExtensions.cs (hunk 28 above)
Comment (Madhan-Reddy-ni) [reply DotNet Examples/TestMonitor/DeleteResults/TestDataManagerDeleteExtensions.cs]: updated the method names as raise post request and raise delete request in the HTTP client file

File: DotNet Examples/TestMonitor/DeleteResults/DeleteResults.cs (hunk 20 above)
Comment (Madhan-Reddy-ni) [reply DotNet Examples/TestMonitor/DeleteResults/DeleteResults.cs]: Updated the message

File: DotNet Examples/TestMonitor/DeleteResults/DeleteResults.cs (hunk 23 above)
Comment (Madhan-Reddy-ni) [reply DotNet Examples/TestMonitor/DeleteResults/DeleteResults.cs]: Updated the message
